src/main/java/ru/itmo/sd/deadliner2bot/state/BaseState.java

```diff
@@ -48,27 +48,42 @@ public void postConstruct() {
     public List<BotApiMethod<?>> process(Chat chat, String message) {
         List<BotApiMethod<?>> response = new ArrayList<>();
         if (commandsInfo.get("show-todos-for-today").testMessageForCommand(message)) {
-            List<Todo> notCompletedTodos = todoService.findNotCompletedTodosByChatId(chat.getChatId(), LocalDateTime.now());
-            String notCompletedTodosText = messageFormatter.notCompletedTodos(notCompletedTodos, stateMessages.getMessageByKey(chatStateEnum, "active-todos-header"), false);
-            response.add(messageUtils.createMessage(chat, notCompletedTodosText));
+            List<Todo> notCompletedTodos =
+                    todoService.findNotCompletedTodosByChatId(chat.getChatId(), LocalDateTime.now());
+            String responseMessage;
+            if (notCompletedTodos.isEmpty()) {
+                responseMessage = stateMessages.getMessageByKey(chatStateEnum, "no-active-todos");
+            } else {
+                responseMessage = messageFormatter.notCompletedTodos(notCompletedTodos,
+                        stateMessages.getMessageByKey(chatStateEnum, "active-todos-header"), false);
+            }
+            response.add(messageUtils.createMessage(chat, responseMessage));
             return response;
         } else if (commandsInfo.get("select-todo").testMessageForCommand(message)) {
-            chat.setState(ChatStateEnum.SELECT_TODO_STATE);
-            chatRepository.save(chat);
-            List<Todo> notCompletedTodos = todoService.findNotCompletedTodosByChatId(chat.getChatId(), LocalDateTime.now());
-            String notCompletedTodosText = messageFormatter.notCompletedTodos(notCompletedTodos, null, true);
-            response.add(messageUtils.createMessage(chat, notCompletedTodosText));
-            response.add(messageUtils.createMessage(chat, stateMessages.getMessageByKey(chatStateEnum, "select-todos-request")));
+            List<Todo> notCompletedTodos =
+                    todoService.findNotCompletedTodosByChatId(chat.getChatId(), LocalDateTime.now());
```

Comment:
I think we need to add logic of selection todos that marked as done. May be user marked something as done accidentely and want to unmark todo.